automation/test-linux.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+
+set -ex
+
+template_name=$1
+namespace="kubevirt"
+
+image_url=""
+#set secret_ref only for rhel OSes
+secret_ref=""
+if [[ $TARGET =~ rhel.* ]]; then
+  image_url="docker://quay.io/openshift-cnv/ci-common-templates-images:${TARGET}"
+  secret_ref="secretRef: common-templates-container-disk-puller"
+else
+  image_url="docker://quay.io/kubevirt/common-templates:${TARGET}"
+fi;
+
+oc apply -n $namespace -f - <<EOF
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: ${TARGET}-datavolume-original
+spec:
+  source:
+    registry:
+      url: "${image_url}"
+      ${secret_ref}
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 30Gi
+EOF
+
+timeout=600
+sample=10
+current_time=0
+while [ $(oc get pods -n $namespace | grep "${TARGET}-datavolume-original.*Running" | wc -l ) -eq 0 ] ; do 
+  oc get pods -n $namespace
+  current_time=$((current_time + sample))
+  if [ $current_time -gt $timeout ]; then
+    error=true
```

Comment:
This `error` variable is set here, but it is ignored and the test continues as if no error happened.